src/refiners/foundationals/latent_diffusion/lora.py

```diff
@@ -223,6 +223,8 @@ def scales(self) -> dict[str, float]:
 
     @staticmethod
     def pad(input: str, /, padding_length: int = 2) -> str:
+        # make all numbers the same length so they sort correctly,
+        # e.g. foo.3.bar -> foo.03.bar
```

Comment:
This should probably be in the docstring

Reply:
Actually I was considering making this method private instead. It's just a helper for `sort_keys` I don't want it in the docs. WDYT?